recipes/textdistance/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "textdistance" %}
+{% set version = "3.0.3" %}
+{% set sha256 = "a938a9b6216076f2b5bed50c56dbeada7db0b0561682466cab72b3a9a1ffa226" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - textdistance
+
+about:
+  home: https://github.com/orsinium/textdistance
+  license: GPL-3.0
+  license_family: GPL3
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
It looks like the `license_file` is included in the source distribution, so `license_file: LICENSE` should be fine. Also delete the license file that's included in the recipe directory.

Reply:
I deleted the license file from the recipe directory and I updated the license_file field. Thank you so much for your feedback and help!!!